<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodesEdges</class>
 <widget class="QDialog" name="NodesEdges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000000</width>
    <height>7000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рёбра и узлы</string>
  </property>
  <widget class="QTableView" name="tv_nodes">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>781</width>
     <height>621</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tv_edges">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>501</width>
     <height>621</height>
    </rect>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Узлы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Рёбра</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="push_add_edge">
      <property name="text">
       <string>Добавить ребро</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_edges">
      <property name="text">
       <string>Редактировать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_del_edge">
      <property name="text">
       <string>Удалить ребро</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>660</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="push_add_node">
      <property name="text">
       <string>Добавить узел</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_nodes">
      <property name="text">
       <string>Редактировать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_del_node">
      <property name="text">
       <string>Удалить узел</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
